tests/test_grounding.py

```diff
@@ -43,3 +41,24 @@ def mock_input(*args):
                 grounder.ground(self.img, self.segments)
         self.assertTrue(self.img.is_grounded())
         self.assertEquals(reconstruct_chars(self.img.ground.classes), "0"*len(self.segments))
+
+    def test_terminal_grounder(self):
+        terminal = TerminalGrounder()
+        segmenter = ContourSegmenter()
+        image = ImageFile('digits1')
+        segments = segmenter.process(image.image)
+        characters = ['9', '8', '2', '1', '4', '8', '0', '8', '6', '5', '1', '3', '2', '8',
```

Comment:
We don't need to be hardcoding/duplicating the character sequence over and over. You can just use the already provided ground data - see how it's working in the previous test